media-libs/freetype/freetype-2.11.0.recipe

```diff
@@ -75,7 +78,7 @@ defineDebugInfoPackage freetype$secondaryArchSuffix \
 BUILD()
 {
 	unset meson
-	meson setup build . --buildtype release \
+	meson setup build . --buildtype release  \
```

Comment:
Extra space added? Shouldn't be needed also.

Reply:
Fixed